src/main/scala/com/codacy/analysis/cli/clients/Credentials.scala

```diff
@@ -46,7 +49,7 @@ object Credentials {
       project <- projectOpt
       userName <- userNameOpt
     } yield {
-      APIToken(apiToken, apiUrlOpt, userName, project)
```

Comment:
Would be nice to use an ArgParser, check https://github.com/codacy/codacy-coverage-reporter/blob/master/src/main/scala/com/codacy/configuration/parser/ConfigurationParser.scala#L63